### PR TITLE
Add support for vision models in OpenAI Chat API, e.g. gpt-4o, gpt-4-turbo, etc.

### PR DESCRIPTION
**Pull request by DavdGao**
## Description

- Update format function to support vision model in OpenAIChatWrapper
- Add new unit tests for vision model format
- Update tutorial
- A new conversation example for gpt-4o and other vision models

## Checklist

Please check the following items before code is ready to be reviewed.

- [x]  Code has passed all tests
- [x]  Docstrings have been added/updated in Google Style
- [x]  Documentation has been updated
- [x]  Code is ready for review

### CODE REVIEW

File: docs/sphinx_doc/en/source/tutorial/206-prompt.md [hunk 1]
```diff
@@ -100,6 +102,75 @@ print(prompt)
...
+
+Note the invalid image urls (e.g. `/Users/xxx/test.mp3`) will be ignored.
+
+```python
+from agentscope.models import OpenAIChatWrapper
+from agentscope.message import Msg
+
+model = OpenAIChatWrapper(
+    config_name="", # empty since we directly initialize the model wrapper
+    model_name="gpt-4",
```
Comment (pan-x-c) [inline docs/sphinx_doc/en/source/tutorial/206-prompt.md:119]: gpt-4 -> gpt-4o

File: docs/sphinx_doc/zh_CN/source/tutorial/206-prompt.md [hunk 2]
```diff
@@ -76,6 +78,75 @@ print(prompt)
...
+
+注意无效的图片url（例如`/Users/xxx/test.mp3`）将被忽略。
+
+```python
+from agentscope.models import OpenAIChatWrapper
+from agentscope.message import Msg
+
+model = OpenAIChatWrapper(
+    config_name="", # 为空，因为我们直接初始化model wrapper
+    model_name="gpt-4",
```
Comment (pan-x-c) [inline docs/sphinx_doc/zh_CN/source/tutorial/206-prompt.md:95]: the same as above

File: docs/sphinx_doc/en/source/tutorial/206-prompt.md (hunk 1 above)
Comment (DavdGao) [reply docs/sphinx_doc/en/source/tutorial/206-prompt.md]: Done

File: docs/sphinx_doc/zh_CN/source/tutorial/206-prompt.md (hunk 2 above)
Comment (DavdGao) [reply docs/sphinx_doc/zh_CN/source/tutorial/206-prompt.md]: Done

File: docs/sphinx_doc/en/source/tutorial/206-prompt.md [hunk 3]
```diff
@@ -100,6 +102,75 @@ print(prompt)
...
+)
+
+prompt = model.format(
+   Msg("system", "You're a helpful assistant", role="system"),   # Msg object
+   [                                                             # a list of Msg objects
+      Msg(name="user", content="Describe this image", role="user", url="https://xxx.png"),
+      Msg(name="user", content="And these images", role="user", urls=["/Users/xxx/test.png", "/Users/xxx/test.mp3"]),
```
Comment (pan-x-c) [inline docs/sphinx_doc/en/source/tutorial/206-prompt.md:126]: urls -> url

File: docs/sphinx_doc/zh_CN/source/tutorial/206-prompt.md [hunk 4]
```diff
@@ -76,6 +78,75 @@ print(prompt)
...
+)
+
+prompt = model.format(
+   Msg("system", "You're a helpful assistant", role="system"),   # Msg 对象
+   [                                                             # Msg 对象的列表
+      Msg(name="user", content="Describe this image", role="user", url="https://xxx.png"),
+      Msg(name="user", content="And these images", role="user", urls=["/Users/xxx/test.png", "/Users/xxx/test.mp3"]),
```
Comment (pan-x-c) [inline docs/sphinx_doc/zh_CN/source/tutorial/206-prompt.md:102]: the same as above

File: src/agentscope/agents/user_agent.py [hunk 5]
```diff
@@ -81,7 +81,9 @@ def reply(
         # Input url of file, image, video, audio or website
         url = None
         if self.require_url:
-            url = input("URL: ")
+            url = input("URL (or Enter to skip): ")
```
Comment (qbc2016) [inline src/agentscope/agents/user_agent.py:84]: how to combine this with as_studio?  @rayrayraykk

File: docs/sphinx_doc/en/source/tutorial/206-prompt.md (hunk 3 above)
Comment (DavdGao) [reply docs/sphinx_doc/en/source/tutorial/206-prompt.md:126]: Done

File: docs/sphinx_doc/zh_CN/source/tutorial/206-prompt.md (hunk 4 above)
Comment (DavdGao) [reply docs/sphinx_doc/zh_CN/source/tutorial/206-prompt.md:102]: Done

**Review by pan-x-c — COMMENTED** [review]
LGTM

**Review by pan-x-c — APPROVED** [review]

**Pull request merged**